editor.planx.uk/src/components/TestEnvironmentBanner.tsx

```diff
@@ -0,0 +1,42 @@
+import ReportIcon from "@mui/icons-material/Report";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import { styled } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import React, { useState } from "react";
+
+import { getHeaderPadding } from "./Header";
+
+const TestEnvironmentWarning = styled(Box)(({ theme }) => ({
+  display: "flex",
+  backgroundColor: theme.palette.background.paper,
+  color: theme.palette.text.primary,
+  justifyContent: "space-between",
+  alignItems: "center",
+  ...getHeaderPadding(theme),
+}));
+
+const TestEnvironmentBanner: React.FC = () => {
+  const [showWarning, setShowWarning] = useState(true);
+
+  const isTestEnvironment = () => !window.location.href.includes(".uk");
```

Comment:
this might be too naive? but i think it should work! 

intended behavior is: 
- do _not_ show on `editor.planx.uk` or a custom domain which should always be `...gov.uk` for now
- do show on `editor.planx.dev`, `...planx.pizza`, and `localhost`

Reply:
I can't see any issues with this - let's go with it 👌